includes/class-wc-siftscience-admin.php

```diff
@@ -30,33 +32,33 @@ public function __construct(
 				WC_SiftScience_Html $html,
 				WC_SiftScience_Logger $logger,
 				WC_SiftScience_Stats $stats ) {
-			$this->_options = $options;
-			$this->_comm    = $comm;
-			$this->html     = $html;
-			$this->_logger  = $logger;
-			$this->_stats   = $stats;
+			$this->options = $options;
+			$this->comm    = $comm;
+			$this->html    = $html;
+			$this->logger  = $logger;
+			$this->stats   = $stats;
 		}
 
 		public function check_api() {
 			// try requesting a non-existent user score and see that the response isn't a permission fail.
-			$response = $this->_comm->get_user_score( '_dummy_' . rand( 1000, 9999 ) );
-			$this->_logger->log_info( '[api check response] ' . json_encode( $response ) );
-			return isset( $response->status ) && ( $response->status === 0 || $response->status === 54 );
+			$response = $this->comm->get_user_score( '_dummy_' . wp_rand( 1000, 9999 ) );
+			$this->logger->log_info( '[api check response] ' . wp_json_encode( $response ) );
+			return isset( $response->status ) && ( 0 === $response->status || 54 === $response->status );
 		}
 
 		public function get_sections() {
 			global $current_section;
-			$sections  = array(
-				'' => 'Settings',
+			$sections = array(
+				''          => 'Settings',
+				'stats'     => 'Stats',
+				'debug'     => 'Debug',
```

Comment:
I think this will change the order of the sub-sections in the Sift settings area. Is this intentional?

Reply:
i don't know i just ordered text ok then what to check